### PR TITLE
fix: tooltip not getting dismissed 

### PR DESCRIPTION
**Pull request by hdz-666**
### Details(context, link the issue, how was the bug fixed, what does the new feature do)

- event propogation is causing this bubble to happen, stop it after event triggered is bubbled and problem will be solved ;
- 
similar to :  https://github.com/radix-ui/primitives/issues/2248
-

### CODE REVIEW

File: packages/roomkit-react/src/Prebuilt/components/MoreSettings/SplitComponents/DesktopOptions.tsx [hunk 1]
```diff
@@ -120,7 +120,12 @@ export const DesktopOptions = ({
         onOpenChange={value => updateState(MODALS.MORE_SETTINGS, value)}
         modal={false}
       >
-        <Tooltip title="More options">
+        <Tooltip
+          title="More options"
+          onFocus={(event: FocusEvent<Element>) => {
+            event.preventDefault();
```
Comment (hdz-666) [inline packages/roomkit-react/src/Prebuilt/components/MoreSettings/SplitComponents/DesktopOptions.tsx:126]: event propogation is causing this bubble to happen, stop it after event is sent is bubbled and problem will be solved : https://github.com/radix-ui/primitives/issues/2248

Comment (raviteja83) [reply packages/roomkit-react/src/Prebuilt/components/MoreSettings/SplitComponents/DesktopOptions.tsx:126]: should we add it at the tooltip level? we can safely call it on every tooltip right? why do we need to pass a fn where we use tooltip?
did we also check most places where tooltip is used? this was just given as example as it is easy to repro

Comment (KaustubhKumar05) [reply packages/roomkit-react/src/Prebuilt/components/MoreSettings/SplitComponents/DesktopOptions.tsx:126]: I think Chat and any other button that toggles something (sidepane, dropdown) and has a tooltip has this issue

Comment (hdz-666) [reply packages/roomkit-react/src/Prebuilt/components/MoreSettings/SplitComponents/DesktopOptions.tsx:126]: ok wil check this @raviteja83  just thought happening here

Comment (KaustubhKumar05) [reply packages/roomkit-react/src/Prebuilt/components/MoreSettings/SplitComponents/DesktopOptions.tsx:126]: 👀

File: packages/roomkit-react/src/Tooltip/Tooltip.tsx [hunk 2]
```diff
@@ -61,13 +61,12 @@ export const Tooltip: React.FC<
...
   disabled = false,
   delayDuration = 200,
-  ...props
 }) => {
   const [open, setOpen] = useState(false);
 
   return (
     <BaseTooltip.Provider>
-      <TooltipRoot delayDuration={delayDuration} open={open && !disabled} onOpenChange={setOpen} {...props}>
+      <TooltipRoot delayDuration={delayDuration} open={open && !disabled} onOpenChange={setOpen}>
```
Comment (raviteja83) [inline packages/roomkit-react/src/Tooltip/Tooltip.tsx:69]: why did we remove the props? are there no other props tooltip takes?

**Review by raviteja83 — APPROVED** [review]

**Review by KaustubhKumar05 — APPROVED** [review]

**Pull request merged**